src/coreclr/nativeaot/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3160,4 +3160,7 @@
   <data name="InvalidOperation_ComInteropRequireComWrapperInstance" xml:space="preserve">
     <value>COM Interop requires ComWrapper instance registered for marshalling.</value>
   </data>
+  <data name="InvalidOperation_InterfaceNotSupported" xml:space="preserve">
```

Comment:
Could you please revert all changes in this file?

Reply:
@jkotas done